src/main/java/org/gridsuite/modification/server/service/BuildWorkerService.java

```diff
@@ -58,21 +58,27 @@ public class BuildWorkerService {
 
     private BuildStoppedPublisherService stoppedPublisherService;
 
+    private BuildFailedPublisherService failedPublisherService;
+
     private Map<String, CompletableFuture<List<ModificationInfos>>> futures = new ConcurrentHashMap<>();
 
     private Map<String, BuildCancelContext> cancelBuildRequests = new ConcurrentHashMap<>();
 
+    private Map<String, BuildFailedContext> failedBuildRequests = new ConcurrentHashMap<>();
```

Comment:
I think this map is not used any more. maybe it should be deleted

Reply:
Fixed